.projenrc.ts

```diff
@@ -62,11 +63,12 @@ const project = new typescript.TypeScriptProject({
     'typescript-json-schema',
   ],
 
-  // we need the compiled .js files for the init tests (we run the cli in there)
-  compileBeforeTest: true,
```

Comment:
Not necessary anymore since init templates are not part of the test command anymore